### ISSUE TITLE
About Loss Function

### ISSUE THREAD

**Issue by FrostyDth** [Loli-Eternally/D2NN-with-Pytorch#1]
In your function 'Dectect_region', you've already normalized your outputs by deviding it with the total, that is to say that, the result of the function adding up to 1, but in your loss function,  why will you choose 0.1*det_labels? If I have a label in one-hot form, and mutiply it with 0.1, I will get [0.1 0 0 0 0 0 0 0 0 0 ] for example, but the output may be like [0.9, 0.05 0.05 0 0 0 0 0 0 0]. But it's strange that using 0.1*det_labels the accuracy is significantly higher. So may you give an explanation on why you choose 0.1*det_label rather than det_label?

**Comment by Loli-Eternally** [comment]
I've got your point. The loss function with the original one-hot form is used to make the highest contrast between the correct and error detectors. The reason why I choose 0.1det_labels, which means [0.1 0 0 0 0 0 0 0 0 0 ] one-hot for loss, is that I consider the probability of the correct detector approximates to 0.1-0.2, and other error detectors have a lower probability than the correct one. In fact, I try to make a trade-off between contrast ratio and model accuracy. @FrostyDth
